notebooks/official/workbench/exploratory_data_analysis/explore_data_in_bigquery_with_workbench.ipynb

```diff
@@ -1748,7 +1684,7 @@
       },
       "source": [
         "## Save some outputs to Cloud Storage\n",
-        "Sometimes you may want some of your code output - tables, graphs, etc. - to persist outside of your notebook. This could be to preserve some results for when you're done with the notebook or to share with various collaborators via other mediums.\n",
+        "Sometimes, you may want some of your code output - tables, graphs, etc. - to persist outside of your notebook. This could be to preserve some results for when you're done with the notebook or to share with various collaborators via other mediums.\n",
```

Comment:
- change: " output - tables, graphs, etc. - to"
- to: " output - for example,  tables and graphs - to"